### PR TITLE
Add script for developers to manually relay messages

### PR DESCRIPTION
**Pull request by boyuan-chen**
:clipboard: Add associated issues, tickets, docs URL here.

## Overview

Describe what your Pull Request is about in a few sentences.

Add the script for developers to manually relay messages

## Changes

Describe your changes and implementation choices. More details make PRs easier to review.

- Use the SDK to check and relay messages

## Testing

Describe how to test your new feature/bug fix and if possible, a step by step guide on how to demo this.

### CODE REVIEW

**Review by souradeep-das — COMMENTED** [review]
🚀

File: boba_community/boba-relayer/README.md [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+> This script supports all BOBA L2s
+
+  ```bash
+  $ git clone git@github.com:bobanetwork/boba.git
+  $ cd boba/boba_community/boba-relayer
+  $ yarn clean # only needed / will only work if you had it installed previously
+  $ yarn
+  ```
+
+  Then, add `.env` in `boba/boba_examples/boba-fee`.
```
Comment (allush) [inline boba_community/boba-relayer/README.md]: incorrect folder. must be `boba/boba_community/boba-relayer`

Comment (boyuan-chen) [reply boba_community/boba-relayer/README.md]: Thanks!

**Comment by InoMurko** [comment]
should we add a reference to this example in the docs?

**Comment by boyuan-chen** [comment]
> should we add a reference to this example in the docs?

Absolutely!

**Review by wsdt — APPROVED** [review]

**Comment by boyuan-chen** [comment]
> ⚠️ **This PR triggers a minor release of the Bundler SDK** (Account Abstraction) to NPM as you made changes in the corresponding `bundler_sdk` directory.
> 
> Please make sure this is intentional.
> 
> The package that is going to be updated is `@bobanetwork/bundler_sdk`.

@wsdt Any idea why this touches bundler sdk? I didn't change anything outside the boba_relayer folder except `.gitignore`?

**Comment by wsdt** [comment]
> > warning **This PR triggers a minor release of the Bundler SDK** (Account Abstraction) to NPM as you made changes in the corresponding `bundler_sdk` directory.
> > Please make sure this is intentional.
> > The package that is going to be updated is `@bobanetwork/bundler_sdk`.
> 
> @wsdt Any idea why this touches bundler sdk? I didn't change anything outside the boba_relayer folder except `.gitignore`?

my bad! The commenter has a minor bug. 
Will fix. 

The actual release pipeline should be correct and not release a new npm package.

**Comment by boyuan-chen** [comment]
Add this to our docs. https://github.com/bobanetwork/boba/pull/797

**Review by InoMurko — APPROVED** [review]

**Review by souradeep-das — APPROVED** [review]
nice!!

**Pull request merged**